README.md

```diff
@@ -1,117 +1,84 @@
 
-Crossplane can be installed into an existing Kubernetes cluster to add managed
-service provisioning or deployed as a dedicated control plane for multi-cluster
-management and workload scheduling.
+Crossplane builds on the Kubernetes control plane. It is composed of [Stacks] -
+easy to install packages of Kubernetes [custom resources and controllers] that
+extend Crossplane with new functionality. A stack can extend Crossplane with the
+ability to manage the infrastructure of a new cloud provider, or to deploy and
+manage a new cloud-native application. This enables Crossplane users to:
 
-Crossplane enables the community to build and publish Stacks to add more clouds
-and cloud services to Crossplane with support for out-of-tree extensibility and
-independent release schedules. Crossplane includes Stacks for [GCP][stack-gcp], 
-[AWS][stack-aws], and [Azure][stack-azure] today.
+1. Install the **infrastructure stacks** for their desired clouds to enable
+   on-demand, **cloud-portable provisioning of infrastructure** services like
+   databases, caches, and Kubernetes clusters.
+1. Define **portable application workloads**, including their cloud
+   infrastructure dependencies, and schedule them to the most appropriate
+   infrastructure across any cloud.
+1. Package their workloads as an **application stack** that others may easily
+   deploy.
 
-<h4 align="center"><img src="docs/media/crossplane-overview.png" alt="Crossplane"></h4>
+The full vision and architecture of the Crossplane project is described in our
+[architecture document].
 
-Crossplane has four main feature areas that can be used independently:
-1. [Crossplane Services](docs/README.md#crossplane-services) - provision managed services from kubectl.
-1. [Crossplane Stacks](docs/README.md#crossplane-stacks) - extend Crossplane with new functionality.
-1. [Crossplane Workloads](docs/README.md#crossplane-workloads) - define complete applications and schedule across
-   clusters, regions, and clouds.
-1. [Crossplane Clusters](docs/README.md#crossplane-clusters) - manage multiple Kubernetes clusters from a single
-   control plane.
+## Get Started
 
-## Architecture and Vision
+Take a look at the [getting started] guide to learn how to install Crossplane,
+install a stack, and provision cloud infrastructure using `kubectl`. Refer to
+the Crossplane [documentation] for comprehensive guides to using Crossplane,
+including deploying an application packaged as a stack.
 
-The full architecture and vision of the Crossplane project is described in depth in the [architecture document](https://docs.google.com/document/d/1whncqdUeU2cATGEJhHvzXWC9xdK29Er45NJeoemxebo/edit?usp=sharing). It is the best place to learn more about how Crossplane fits into the Kubernetes ecosystem, the intended use cases, and comparisons to existing projects.
+## Get Involved
 
-## Getting Started and Documentation
+* Discuss Crossplane on [Slack] or our [developer mailing list].
```

Comment:
I think we would still be responsive if someone were to utilize our mailing list, but it has not had a post since 2018. Do we still want to direct people there? If so, it might be worth cross-posting announcements and releases on the mailing list as well. I know there is still a large group of people that prefer mailing lists to github / slack communication.

Reply:
I'm happy either way.

Reply:
Agreed, would not block on this!